examples/Utils/Helper.php

```diff
@@ -42,4 +42,26 @@ public static function getShortPrintableDatetime(): string
     {
         return (new DateTime())->format("mdHisv");
     }
+
```

Comment:
### 2 bugs!

try the function microToBase with the number **2246197**
its return **2**, but i need get **2.25** (GoogleAPI works with 2 decimals)


my function do it well!
```
    public static function microToBase($number) {
        return (float) number_format($number / 1000000, 2, '.', '');
    }
```
try the function baseToMicro with the number **0.1**
its return **0**, but i need get **100000** (GoogleAPI works with 2 decimals- [if you try more ,you get error] ) 


my function do it well!
```
    public static function baseToMicro($number) {//min 10000
        return (int) str_replace('.', '', number_format($number, 2, '.', '') . '0000');
    }
```
i understand my code it's not alginitic , but it works well

Reply:
Thanks for checking, I am adding unit tests and will update the logic accordingly.

Reply:
Google Ads API uses micro unit integers, not floating decimal. I upgraded the methods to:
- Take both float and int amounts as input
- Return float with `microToBase`

This should fix both issues.